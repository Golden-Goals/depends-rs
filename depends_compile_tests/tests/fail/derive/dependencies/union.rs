use depends::derives::Dependencies;

#[derive(Dependencies)]
union Bar {
    node1: usize,
    node2: usize,
}

fn main() {}
