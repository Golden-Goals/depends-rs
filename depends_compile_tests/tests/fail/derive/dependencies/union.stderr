error: expected `struct`
 --> tests/fail/derive/dependencies/union.rs:4:1
  |
4 | union Bar {
  | ^^^^^
